src/main/webapp/app/tasks/dialog/task-due-date-dialog/task-due-date-dialog.component.ts

```diff
@@ -0,0 +1,47 @@
+import {Component, EventEmitter, Inject, Output, ViewChild} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import {ThemePalette} from '@angular/material/core';
+import {MatDialogRef, MAT_DIALOG_DATA} from '@angular/material/dialog';
+import { Common } from '../../../Utility/common';
+/**
+ * A dialog created for selection of date and time 
+ */
+@Component({
+  selector: 'tiny-task-due-date-dialog',
+  templateUrl: './task-due-date-dialog.component.html',
+  styleUrls: ['./task-due-date-dialog.component.scss']
+})
+export class TaskDueDateDialogComponent {
+
+  @ViewChild('picker') picker: any;
```

Comment:
Why are you defining it even though it is not used?

Reply:
Yes right this should be removed